drivers/gles3/storage/light_storage.h

```diff
@@ -141,7 +141,7 @@ struct ReflectionAtlas {
 	int size = 0;
 
 	int mipmap_count = 1; // number of mips, including original
-	int mipmap_size[8];
+	int mipmap_size[8] = {};
```

Comment:
Fixes this:
![image](https://github.com/godotengine/godot/assets/4701338/a1500b77-bfa1-4a79-b061-e8e119fa35b9)

CC @clayjohn @BastiaanOlij 

There's a bunch more issues with uninitialized stuff in rendering code, would be good for the rendering team to make a pass on the reports and decide whether to init all the structs properly, or what other approach to take. We've had a number of bugs on some platforms/GPUs due to uninitialized struct members.